lib/services/cordova-migration-service.ts

```diff
@@ -107,8 +117,25 @@ export class CordovaMigrationService implements ICordovaMigrationService {
 			});
 
 			let supportedPlugins = this.pluginsForVersion(toVersion).wait();
-			plugins = _.filter(plugins, plugin => _.contains(supportedPlugins, plugin));
+			plugins = _.filter(plugins, plugin => {
+				if (plugin.indexOf('@') > -1) {
+					let pluginBasicInfo = this.$pluginsService.getPluginBasicInformation(plugin);
+					if (this.isPluginSupported(pluginBasicInfo.name, pluginBasicInfo.version, toVersion)) {
+						supportedPlugins.push(plugin);
+					} else if (!_.find(this.removedPluginsCache, p => plugin)) {
+						let remove = `Remove ${plugin} from all configuration`;
+						let cancel = 'Cancel Cordova migration';
+						let choice = this.$prompter.promptForChoice(`Plugin ${plugin} is not supported for Cordova version ${toVersion} what do you want to do?`, [remove, cancel]).wait();
+						if (choice === cancel) {
+							this.$errors.failWithoutHelp("Cordova migration interrupted"); 
+						} else {
+							this.removedPluginsCache.push(plugin);
+						}
+					}
+				}
 
+				return _.contains(supportedPlugins, plugin);
```

Comment:
Shouldn't we warn the users when we remove some of the CorePlugins (not the marketplace ones)? It looks like we are silently removing the plugins.